darts/models/regression_ensemble_model.py

```diff
@@ -0,0 +1,93 @@
+"""
+Regression ensemble model
+-----------------------
+"""
+from sklearn.linear_model import LinearRegression
+from typing import Optional, List
+
+from darts.timeseries import TimeSeries
+from darts.models import EnsembleModel, StandardRegressionModel
+from darts.models.forecasting_model import ForecastingModel
+from darts.models.torch_forecasting_model import TorchForecastingModel
+from darts.logging import get_logger, raise_if, raise_if_not
+
+logger = get_logger(__name__)
+
+
+class RegressionEnsembleModel(EnsembleModel):
+    def __init__(self,
+                 forecasting_models: List[ForecastingModel],
+                 regression_train_n_points: int,
+                 regression_model=LinearRegression(n_jobs=-1, fit_intercept=False)):
+        """
+        Class for ensemble models using a regression model for ensembling individual models' predictions
+        The provided regression model must implement fit() and predict() methods
+        (e.g. scikit-learn regression models)
+
+        Parameters
+        ----------
+        forecasting_models
+            List of forecasting models whose predictions to ensemble
+        regression_train_n_points
+            The number of points to use to train the regression model
+        regression_model
+            Any regression model with predict() and fit() methods (e.g. from scikit-learn)
+            Default: `sklearn.linear_model.LinearRegression(n_jobs=-1, fit_intercept=False)`
+        """
+        super().__init__(forecasting_models)
+
+        # wrap provided regression_model in a StandardRegressionModel (if not already the case)
+        if isinstance(regression_model, StandardRegressionModel):
+            # raise exception if train_n_points value is ambiguous
+            model_train_n_points = regression_model.train_n_points
+            raise_if(model_train_n_points is not None and regression_train_n_points != model_train_n_points,
+                     "Provided StandardRegressionModel.train_n_points parameter doesn't match specified"
+                     " regression_train_n_points parameter.",
+                     logger)
+
+            regression_model.train_n_points = regression_train_n_points
+        else:
+            regression_model = StandardRegressionModel(regression_train_n_points, regression_model)
+
+        self.regression_model = regression_model
+
+    def fit(self, training_series: TimeSeries, target_series: Optional[TimeSeries] = None) -> None:
+        # TODO: Factor this out, same logic is used in 3 places already. Need to find an appropiate name
+        if target_series is None:
+            target_series = training_series
+        raise_if_not(all(training_series.time_index() == target_series.time_index()),
+                     "training and target series must have same time indices.",
+                     logger)
```

Comment:
If anyone has a suggestion for this, I'd like to hear it. I couldn't come up with a nice name. Best I could do was `get_train_target()`, but …

Note a method like this existed in an earlier version of `MultivariateForecastingModel` and was called `make_fitable_series()` which I didn't understand well (and since it was only used in one place, I removed it, but now need it back).

Reply:
`target_series` will likely be removed by #231 , so I won't be addressing this here anymore.